rbc/typesystem.py

```diff
@@ -456,6 +475,32 @@ def is_complete(self):
             raise NotImplementedError(repr(self))
         return True
 
+    @property
+    def is_concrete(self):
+        """Return True when the Type instance is concrete.
+        """
+        if self.is_atomic:
+            return (self.is_int or self.is_uint or self.is_float
+                    or self.is_complex or self.is_bool
+                    or self.is_string or self.is_char)
+        elif self.is_pointer:
+            return self[0].is_concrete
+        elif self.is_struct:
+            for m in self:
+                if not m.is_concrete:
+                    return False
+        elif self.is_function:
+            if not self[0].is_concrete:
+                return False
+            for a in self[1]:
+                if not a.is_concrete:
+                    return False
+        elif self.is_void:
+            pass
```

Comment:
Shouldn't this return something?

Reply:
The last return statement returns True. The same applies for is_struct and is_function if blocks.